lib/handlers/command/slack/LinkRepo.ts

```diff
@@ -115,7 +114,7 @@ export class LinkRepo implements HandleCommand {
             return ctx.messageClient.respond(err)
                 .then(() => Success, failure);
         }
-        if (!isChannel(this.channelId)) {
+        if (this.channelType !== "channel") {
```

Comment:
What are possible values here? 

 * `channel`

Reply:
`channel` or `user`. Would you rather I make an enum?